app.js

```diff
@@ -0,0 +1,21 @@
+const express = require('express');
+const {
+  getCategories
+} = require('./controllers/games.js');
+
+const app = express();
+app.use(express.json());
+
+app.get('/api/categories', getCategories);
+
+
+app.all('/*', (req, res) => {
+  res.status(404).send({ msg: 'Route not found' });
+});
```

Comment:
This is great - do you have a test for this?

Reply:
nope but I can add one!

Reply:
added a test for the 404 error